doc/concepts.qbk

```diff
@@ -203,29 +203,29 @@ the generator, for example to re-run a test suite at a later time.]
 
 A quasi-random number generator is a __UniformRandomNumberGenerator which
 provides a deterministic sequence of quasi-random numbers, based on some
-algorithm and internal state. [classref boost::random::niederreiter_base2
+algorithm and internal state. [classref boost::random::niederreiter_base2_engine
 Niederreiter Base 2] generator is an example of such a [qrng quasi-random
 number generator]. The "quasi" modifier is used to denote more clearly that the
 values produced by such a generator are neither random nor pseudo-random, but
-they form a low discrepancy sequence. The intuitive idea is that a low discrepancy 
+they form a low discrepancy sequence. The intuitive idea is that a low discrepancy
 sequence is more evenly distributed than a pseudo random sequence would be.
 For example, if we generate a low discrepancy sequence of 2D points on a square,
 this square would be covered more evenly, and the number of points falling to any
 part of the square would be proportional to the number of points in the whole square.
-Such sequences share some properties of  random variables and in certain applications 
+Such sequences share some properties of  random variables and in certain applications
 such as the quasi-Monte Carlo method  their lower discrepancy is an important advantage.
 
-[note Quasi-random sequences are known to give efficient numerical integration 
-rules in many Bayesian statistical problems where the posterior distribution can be 
+[note Quasi-random sequences are known to give efficient numerical integration
+rules in many Bayesian statistical problems where the posterior distribution can be
 transformed into periodic functions on a hypercube.]
```

Comment:
I do not believe there is any requirement that the function must be periodic.

Reply:
I didn't imply that periodicity was a requirement. I just meant that QMC numerical integration is notably efficient there. But I think that we can broaden the description and make it less technical. 

For example:
Quasi-Monte Carlo method uses a low-discrepancy sequence such as the Niederreiter base 2 sequence, the Sobol sequence, or the Faure sequence among the others. The advantage of using low-discrepancy sequences is a probabilistically faster rate of convergence. Quasi-Monte Carlo has a rate of convergence around to O(1/N), whereas the rate for the Monte Carlo method is O(N^−(1/2)).

Reply:
Oh, I didn't know that there was a better convergence rate for periodic integrands using qmc. If you do add this to the docs, make sure to get the logarithmic factors on the convergence: O(log(N)^d/N).

Reply:
Nah, rewrote the note like this

[note Quasi-Monte Carlo method uses a low-discrepancy sequence such as the 
Niederreiter base 2 sequence, the Sobol sequence, or the Faure sequence among the others. 
The advantage of using low-discrepancy sequences is a probabilistically faster rate of 
convergence. Quasi-Monte Carlo has a rate of convergence close to '''<code>O(1/N)</code>''', 
whereas the rate of convergence for the Monte Carlo method, which uses a pseudo-random sequence, 
is '''<code>O(N<superscript>-0.5</superscript>)</code>'''.] 

I think "close to" is enough here.